ConwaysGameOfLife.Tests/CellShould.cs

```diff
@@ -38,18 +38,16 @@ public void WhenInitialized_SetLocation(int x, int y)
             
             Assert.Equal(new Location(x, y), result);
         }
-        [Fact]
-        public void WhenCellStateAndLocationAreSame_BeEqual()
+
+        [Theory]
+        [InlineData(0, 1, CellState.Dead, 0, 1, CellState.Dead, true)]
+        [InlineData(0, 1, CellState.Alive, 0, 1, CellState.Dead, false)]
+        public void WhenCellStateAndLocationAreSame_BeEqual(int x1, int y1, CellState oneState, int x2, int y2, CellState twoState, bool expected) // TODO: rename method

```

Comment:
Let's tidy up the naming to be consistent (using both `one`/`two` and `1`/`2`). Additionally, the order of the params could be consistent with the signature of the `Cell` constructor:

```suggestion
        [Theory]
        [InlineData(CellState.Dead, 0, 1, CellState.Dead, 0, 1, true)]
        [InlineData(CellState.Alive, 0, 1, CellState.Dead, 0, 1, false)]
        public void WhenCellStateAndLocationAreSame_BeEqual(CellState state1, int x1, int y1, CellState state2, int x2, int y2, bool expected) // TODO: rename method
```

Reply:
sorry late to the party but can also look at using **member data** to allow you to use objects for inline data